### PR TITLE
GSAGH-261 refactor google style guide

### PR DESCRIPTION
**Pull request by kpne**

### CODE REVIEW

File: GsaGH/Parameters/2_Geometry/GsaElement1dGoo.cs [hunk 1]
```diff
@@ -18,30 +18,36 @@ public class GsaElement1dGoo : GH_OasysGeometricGoo<GsaElement1d> {
 
     public GsaElement1dGoo(GsaElement1d item) : base(item) { }
 
-    internal GsaElement1dGoo(GsaElement1d item, bool duplicate) : base(null)
-                                      => Value = duplicate
-        ? item.Duplicate()
-        : item;
+    internal GsaElement1dGoo(GsaElement1d item, bool duplicate) : base(null) {
+      Value = duplicate
+                                                                                     ? item.Duplicate()
```
Comment (DominikaLos) [inline GsaGH/Parameters/2_Geometry/GsaElement1dGoo.cs:23]: broken indentation?

File: GsaGH/Parameters/2_Geometry/GsaElement2dGoo.cs [hunk 2]
```diff
@@ -21,32 +21,38 @@ public class GsaElement2dGoo : GH_OasysGeometricGoo<GsaElement2d>,
...
 
-    internal GsaElement2dGoo(GsaElement2d item, bool duplicate) : base(null)
-                                      => Value = duplicate
-        ? item.Duplicate()
-        : item;
+    internal GsaElement2dGoo(GsaElement2d item, bool duplicate) : base(null) {
+      Value = duplicate
+                                                                                     ? item.Duplicate()
+                                                                                     : item;
+    }
```
Comment (DominikaLos) [inline GsaGH/Parameters/2_Geometry/GsaElement2dGoo.cs:28]: broken indentation?

File: GsaGH/Parameters/2_Geometry/GsaMember1dGoo.cs [hunk 3]
```diff
@@ -21,23 +21,28 @@ public class GsaMember1dGoo : GH_OasysGeometricGoo<GsaMember1d> {
 
     public GsaMember1dGoo(GsaMember1d item) : base(item) { }
 
-    internal GsaMember1dGoo(GsaMember1d item, bool duplicate) : base(null)
-                                      => Value = duplicate
-        ? item.Duplicate()
-        : item;
+    internal GsaMember1dGoo(GsaMember1d item, bool duplicate) : base(null) {
+      Value = duplicate
+                                                                                   ? item.Duplicate()
```
Comment (DominikaLos) [inline GsaGH/Parameters/2_Geometry/GsaMember1dGoo.cs:26]: broken indentation?

File: GsaGH/Parameters/2_Geometry/GsaMember1dParameter.cs [hunk 4]
```diff
@@ -31,9 +31,10 @@ public override string TypeName
     public override void DrawViewportMeshes(IGH_PreviewArgs args) {
     }
 
-    protected override GsaMember1dGoo PreferredCast(object data)
-      => data.GetType() == typeof(GsaMember1d)
-        ? new GsaMember1dGoo((GsaMember1d)data)
-        : base.PreferredCast(data);
+    protected override GsaMember1dGoo PreferredCast(object data) {
+      return data.GetType() == typeof(GsaMember1d)
+                                                                         ? new GsaMember1dGoo((GsaMember1d)data)
```
Comment (DominikaLos) [inline GsaGH/Parameters/2_Geometry/GsaMember1dParameter.cs:36]: broken indentation?

File: GsaGH/Parameters/2_Geometry/GsaMember2dGoo.cs [hunk 5]
```diff
@@ -24,23 +24,28 @@ public class GsaMember2dGoo : GH_OasysGeometricGoo<GsaMember2d>,
 
     public GsaMember2dGoo(GsaMember2d item) : base(item) { }
 
-    internal GsaMember2dGoo(GsaMember2d item, bool duplicate) : base(null)
-                                      => Value = duplicate
-        ? item.Duplicate()
-        : item;
+    internal GsaMember2dGoo(GsaMember2d item, bool duplicate) : base(null) {
+      Value = duplicate
+                                                                                   ? item.Duplicate()
```
Comment (DominikaLos) [inline GsaGH/Parameters/2_Geometry/GsaMember2dGoo.cs:29]: broken indentation?

File: GsaGH/Parameters/2_Geometry/GsaMember3d.cs [hunk 6]
```diff
@@ -181,20 +188,24 @@ public class GsaMember3d {
...
     }
 
-    private void UpdatePreview()
-      => Display.PreviewMem3d(ref _mesh,
-        ref _previewHiddenLines,
-        ref _previewEdgeLines,
-        ref _previewPts);
+    private void UpdatePreview() {
+      Display.PreviewMem3d(ref _mesh,
+                                         ref _previewHiddenLines,
```
Comment (DominikaLos) [inline GsaGH/Parameters/2_Geometry/GsaMember3d.cs:206]: broken indentation?

File: GsaGH/Parameters/2_Geometry/GsaNode.cs [hunk 7]
```diff
@@ -253,7 +259,10 @@ public bool IsSupport
 
     public void UpdateUnit(LengthUnit unit) {
       if (unit == LengthUnit.Meter) // convert from meter to input unit if not meter
+{
```
Comment (DominikaLos) [inline GsaGH/Parameters/2_Geometry/GsaNode.cs:262]: broken indentation?

File: GsaGH/Parameters/2_Geometry/GsaNode.cs [hunk 8]
```diff
@@ -282,7 +291,9 @@ public bool IsSupport
       };
 
       if ((Color)_node.Colour != Color.FromArgb(0, 0, 0)) // workaround to handle that Color is non-nullable type
+{
```
Comment (DominikaLos) [inline GsaGH/Parameters/2_Geometry/GsaNode.cs:294]: as above

File: GsaGH/Parameters/2_Geometry/GsaNodeGoo.cs [hunk 9]
```diff
@@ -23,19 +23,22 @@ public class GsaNodeGoo : GH_OasysGeometricGoo<GsaNode>,
 
     public GsaNodeGoo(GsaNode item) : base(item) { }
 
-    internal GsaNodeGoo(GsaNode item, bool duplicate) : base(null)
-                                      => Value = duplicate
-        ? item.Duplicate()
-        : item;
+    internal GsaNodeGoo(GsaNode item, bool duplicate) : base(null) {
+      Value = duplicate
+                                                                           ? item.Duplicate()
```
Comment (DominikaLos) [inline GsaGH/Parameters/2_Geometry/GsaNodeGoo.cs:28]: broken indentation?

File: GsaGH/Parameters/4_Analysis/GsaAnalysisTask.cs [hunk 10]
```diff
@@ -60,25 +56,28 @@ public enum AnalysisType {
...
-          : ""
-          + " '"
-          + Name
-          + "' "
-          + Type.ToString()
-            .Replace("_", " ")).Trim()
-        .Replace("  ", " ");
+    public override string ToString() {
+      return (Id > 0
+                                                ? "ID:" + Id
```
Comment (DominikaLos) [inline GsaGH/Parameters/4_Analysis/GsaAnalysisTask.cs:72]: broken indentation?

File: GsaGH/Parameters/5_Results/GsaResultParameter.cs [hunk 11]
```diff
@@ -28,9 +28,10 @@ public override string TypeName
       CategoryName.Name(),
       SubCategoryName.Cat9())) { }
 
-    protected override GsaResultGoo PreferredCast(object data)
-      => data.GetType() == typeof(GsaResult)
-        ? new GsaResultGoo((GsaResult)data)
-        : base.PreferredCast(data);
+    protected override GsaResultGoo PreferredCast(object data) {
+      return data.GetType() == typeof(GsaResult)
+                                                                       ? new GsaResultGoo((GsaResult)data)
```
Comment (DominikaLos) [inline GsaGH/Parameters/5_Results/GsaResultParameter.cs:33]: broken indentation?

File: IntegrationTests/3_Components/GlobalPerformance_ModalTests.cs [hunk 12]
```diff
@@ -55,9 +55,9 @@ public class GlobalPerformance_ModalTests {
       GH_Document doc = Document;
       IGH_Param param = Helper.FindParameter(doc, "ModalStiffnesses");
       var output1 = (GH_Number)param.VolatileData.get_Branch(0)[0];
-      var output2 = (GH_Number)param.VolatileData.get_Branch(0)[1];
-      var output3 = (GH_Number)param.VolatileData.get_Branch(0)[2];
-      var output4 = (GH_Number)param.VolatileData.get_Branch(0)[3];
+      //var output2 = (GH_Number)param.VolatileData.get_Branch(0)[1];
```
Comment (DominikaLos) [inline IntegrationTests/3_Components/GlobalPerformance_ModalTests.cs:58]: as above

File: IntegrationTests/3_Components/GlobalPerformance_ModalTests.cs [hunk 13]
```diff
@@ -40,9 +40,9 @@ public class GlobalPerformance_ModalTests {
       GH_Document doc = Document;
       IGH_Param param = Helper.FindParameter(doc, "ModalMasses");
       var output1 = (GH_Number)param.VolatileData.get_Branch(0)[0];
-      var output2 = (GH_Number)param.VolatileData.get_Branch(0)[1];
-      var output3 = (GH_Number)param.VolatileData.get_Branch(0)[2];
-      var output4 = (GH_Number)param.VolatileData.get_Branch(0)[3];
+      //var output2 = (GH_Number)param.VolatileData.get_Branch(0)[1];
```
[the remark this answers is not in the document]
Comment (kpne) [reply IntegrationTests/3_Components/GlobalPerformance_ModalTests.cs:43]: this line is commented out as it awaits a bugfix in GsaAPI, see comment later where the `output2` variable is used

**Review by tlmnrnhrdt — APPROVED** [review]

**Pull request merged**